app/assets/js/rtMedia.backbone.js

```diff
@@ -991,6 +991,43 @@ jQuery( document ).ready( function( $ ) {
 	 * End of issue 124 fix
 	 */
 
+	/**
+	 * UI changes for buddypress nouveau theme on activity page
+	 */
+
+	if( bp_template_pack && 'legacy' !== bp_template_pack ) {
+
+		let whats_new_form = jQuery( '#whats-new-form' );
+
+		jQuery( '#whats-new' ).on( 'focus', function () {
+
+			let rt_uploader_div = jQuery( '#whats-new-form .rtmedia-uploader-div' );
```

Comment:
You already have `#whats_new_form` reference, use it's `find()` method to find sub-elements. 
Please do this at all the places.